### DISCUSSION TITLE
Media Content Page en BF

### DISCUSSION THREAD

**Discussion by AnaMontes11** [Telefonica/mistica-design#1422] [Proposals]
### Goal

Crea página "Media Content" en Brand Factory

### Context

Al incluir la documentación de las Naked Card en BF veo que en cada tipología repetimos la parte de Media Content.
Para evitar crear documentación demasiado extensa, creo que podría estar bien tener una sección específica "Media Content" que hable de Imagen (tipos de aspect ratio, formatos soportados y calidad de imágenes), a la vez hacer lo mismo e incluir documentación específica para video. De esta manera podemos enlazar a esta página y no duplicar info.

### Impact

Mejora en el consumo de la documentación agilizando a la vez su mantenimiento.

### Discussion Questions

..

**Comment by yceballost** [comment]
Yo lo sugerí hace tiempo y me lo tumbastéis!! tener una página "Media" donde se explique el componente imagen y video

De hecho en esta pagina hay algunos ejemplos que estaban en esa pagina 
https://brandfactory.telefonica.com/document/1846#/archive

**Reply by AnaMontes11** [reply]
Pues yo le veo todo el sentido. Cuenta con mi espada!
